File: POO/J/OzCorp/src/br/com/ozcorp/business/Analyst.java
package br.com.ozcorp.business;
public class Analyst extends Employee {
	public Analyst(String name, String rg, long cpf, String enrollment, String email, String password,
			BloodType blood, Gender gender) {
		super(name, rg, cpf, enrollment, email, password, Role.Analyst, blood, gender);
	}
}

File: POO/CS/OzCorp/OzCorp/Business/Analyst.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzCorp.Business
{
    public class Analyst : Employee
    {
        public Analyst(string Name, string RG, long CPF, string Enrollment, string Email, string Password, BloodType Blood, Gender Gender) : base(Name, RG, CPF, Enrollment, Email, Password, Role.GetRole(Roles.Analyst), Blood, Gender)
        {
        }
    }
}
